dbee/adapters/databricks.go

```diff
@@ -0,0 +1,70 @@
+package adapters
+
+import (
+	"database/sql"
+	"fmt"
+	"net/url"
+
+	_ "github.com/databricks/databricks-sql-go"
+
+	"github.com/kndndrj/nvim-dbee/dbee/core"
+	"github.com/kndndrj/nvim-dbee/dbee/core/builders"
+)
+
+// Register client
+func init() {
+	_ = register(&Databricks{}, "databricks")
+}
```

Comment:
There seems to be something wrong in loading this adapter. I specify type `databricks`, but get this error:
```
[core]: failed registering source: persistence.json function DbeeCreateConnection[1]..remote#define#request, line 2: Vim(let):Error invoking '0:function:DbeeCreateConnection' on ch
annel 3:                                                                                                                                                                            
adapters.NewConnection: Mux.GetAdapters: no driver registered for provided type alias  
```
I have loaded using `lazy` as such:
```lua
    {
    	"MattiasMTS/nvim-dbee",
        event = "VeryLazy",
        -- "kndndrj/nvim-dbee",
        branch = "mattias/databricks-adapter",
        dependencies = {
            "MunifTanjim/nui.nvim",
        },
        build = function()
            require("dbee").install()
        end,
    },
```
and default `setup`.

Am i missing something here?

> Note that other adapters works as intended.

Reply:
Change to:

```lua
build = function()
    require("dbee").install("go")
end
```

to force installing via go. Basically because the default will do a lookup (see docs) and binaries don't exist for all branches. So, installing a specific branch require you to do it in a developer way -> which is using go.

Reply:
Awesome, thanks it works great now!

Reply:
Awesome, thanks it works great now!